codegen-core/common-test-models/naming-obstacle-course-prelude-ops.smithy

```diff
@@ -0,0 +1,92 @@
+$version: "1.0"
+namespace crate
+
+use smithy.test#httpRequestTests
+use smithy.test#httpResponseTests
+use aws.protocols#awsJson1_1
+use aws.api#service
+use smithy.framework#ValidationException
+
+/// Confounds model generation machinery by using operations named after every item in the Rust prelude
+@awsJson1_1
+@service(sdkId: "Config")
+service Config {
```

Comment:
These are the kinds of tests that I think are better done in `{client,server}IntegrationTests`. In Kotlin we can easily code generate Smithy models that more exhaustively test this e.g. union / enum shapes with names of types from the prelude are not covered.

Reply:
That's a great idea. I refactored them into Kotlin unit tests in 49d10aecb5d768cece726b2aaab87a1136852040 and also added them to the server codegen test suite.